### ISSUE TITLE
{ videoId: id } search doesn't recognise "#" in title

### ISSUE THREAD

**Issue by StelFoog** [talmobi/yt-search#50]
I've noticed that the resulting video object from `yt-search({ videoId: id})` removes any part of the title at and after a `#`-symbol.

When running this code
```js
const yts = require('yt-search');

main();

async function main() {
	const videoIdSearch = await yts({ videoId: process.argv[2] });
	const res = await yts(process.argv[2]);

	console.log(videoIdSearch.videoId === res.videos[0].videoId);
	console.log(videoIdSearch.title);
	console.log(res.videos[0].title);
}
```

We get the result:
```
user@system $ node test z95fi3uazYA

true
Dragon Ball Z Abridged: Episode 60 - Part 3 -
Dragon Ball Z Abridged: Episode 60 - Part 3 - #DBZA60 | Team Four Star (TFS)
```

**Comment by talmobi** [comment]
Hello there,

Thanks for the report!

Should be fixed in latest just-released patch `2.7.5`

**Comment by StelFoog** [comment]
Working.
